hosting/src/actions/authAction.ts

```diff
@@ -0,0 +1,20 @@
+"use server";
+
+import {cookies} from "next/headers";
+import {redirect} from "next/navigation";
+import {SIGN_IN_ROUTE, SESSION_COOKIE_NAME} from "@/constants";
+
+export async function createSession(token: string) {
```

Comment:
Could you please link me to the React+Firebase documentation for this 🙏

Reply:
I haven't read all the documentation on nextjs, I just fixed a little of @detautama PR https://github.com/oddbit/tanam/pull/346 regarding these changes. Maybe it's not perfect but I would love to read best practices from @detautama  regarding these changes

Reply:
It is [Session Management in Server Actions](https://dev.to/yutakusuno/nextjs14-firebase-authentication-with-google-sign-in-using-cookies-middleware-and-server-actions-48h4#:~:text=Session%20Management%20in%20Server%20Actions). This is the documentation about [Server Actions](https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations)

Reply:
Basically, server actions are used for operations on the server side (fetch data, authentication, etc), but in practice they can also be called in the client component

Reply:
Thank you @detaoddbit what happens if you remove that? I find it peculiar that next requires an additional session handling on top of Firebase Authentication.